Sources/PCCurrentUser.swift

```diff
@@ -81,7 +81,7 @@ public final class PCCurrentUser {
         self.avatarURL = avatarURL
         self.customData = customData
         self.instance = instance
-        self.v5Instance = v5Instance
+        self.v6Instance = v5Instance
```

Comment:
more v5

Reply:
Stupid Xcode..I used the refactor thing where it should change everything..but I don't think it quite worked..